### DISCUSSION TITLE
Custom settings for each game

### DISCUSSION THREAD

**Discussion by amintikk** [skyline-emu/skyline#1420] [Ideas]
Hello, you could put an option to have a custom configuration for each game, for example to be able to activate the "Dock Mode" in Cuphead but have it deactivated in FIFA 22, without having to activate or deactivate it, since it is a bit cumbersome.
**((Google translate))**

Hola, podriais poner una opcion para tener una configuracion personalizada para cada juego, por ejemplo poder activar el "Modo Dock" en Cuphead pero tenerlo desactivado en FIFA 22, sin tener que ir activandolo o desactivandolo todo el rato, ya que es un un poco engorroso.
**((Original))**

**Comment by yhak22** [comment]
That will be awesome .
And if there is a ingame setting where you can change between docked and undocked mode
